miso-web/src/main/webapp/scripts/lims.js

```diff
@@ -317,6 +317,67 @@ var Utils = Utils
               showButtonPanel: true,
             });
             break;
+          case 'compare':
+            input = document.createElement('DIV');
+            var compareTypeControl = document.createElement('SELECT');
+            var values = [{
+              label: 'ignore',
+              comparator: function(x, y) {
+                return true;
+              }
+            }, {
+              label: '>',
+              comparator: function(x, y) {
+                return x > y;
+              }
+            }, {
+              label: '>=',
+              comparator: function(x, y) {
+                return x >= y;
+              }
+            }, {
+              label: '=',
+              comparator: function(x, y) {
+                return x = y;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "x" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)

Reply:
Should be `==`